### ISSUE TITLE
`adb shell input tap` command is not working properly on WSA

### ISSUE THREAD

**Issue by Nova-Noir** [microsoft/WSA#205]
### Steps to reproduce

I have confirmed that no other applications were running in the foreground when I tested it.

`adb shell input tap` command has no effect when running  ( Or I guess that it ran on other application or window? )

1. I opened `Pointer Location` in `Developer options` to make it more clear ( This is an optional step )
2. Then, I closed `development settings` and opened my application ( In my case, `Arknights` )
3. I find the pos I want to tap ( in this case, `X: 1572,  Y: 1114` )
![pos_to_tap](https://user-images.githubusercontent.com/68760718/211750263-cab456c2-0b95-4379-9c66-db1cf70cd15f.png)
4. I ran my command `.\adb.exe -s 127.0.0.1:58526 shell input tap 1572 1114` and nothing happened.
![command_is_not_working](https://user-images.githubusercontent.com/68760718/211750576-39dab89a-7037-4bcd-bac9-fc9369ce2b99.png)
5. I tried it in fullscreen mode and got the same result.
6. I found an icon to resize the resolution when get into fullscreen mode
![image](https://user-images.githubusercontent.com/68760718/211751297-1935caac-a872-4b6d-a748-71f0213db546.png)
7. I pressed it and my application restarted, then, somehow, `adb shell input tap` was working again.
8. I exited fullscreen mode, `adb shell input tap` was not working.
9. I pressed the icon again in windowed mode, the application restarted, but `adb shell input tap` was still not working.
![image](https://user-images.githubusercontent.com/68760718/211752211-ad3f10ee-81bc-4d66-8440-eb33cf0e12a9.png)
10. I tested other applications like `Developer options` and the command is still not working.

To summarize, `adb shell input tap` command only works when my application is running in fullscreen mode with `tap refresh to attempt button` clicked.

I guess that there could be some kind of `focus` so my command just tapped other window?

### ✔️ Expected Behavior

The menu of `任务` should be opened

### ❌ Actual Behavior

The menu was not opened and I can see the data of `Pointer Location` is not the same.

### Please specify the version of Windows Subsystem for Android

2211.40000.11.0

**Comment by Nova-Noir** [comment]
It works when I enabled this feature.

But the frames of the game become extremely low somehow.

**Comment by Nova-Noir** [comment]
Yes It works!

What is the reason caused this problem?

**Comment by Nova-Noir** [comment]
But I also noticed that my game frames become extremely low when this feature enabled.

10fps or so I guess..

**Comment by jaholme** [comment]
Focus changes happen when switching away from different windows. You will see the same behavior on an Android device using freeform windows and switching between apps.
